src/schema/objects/common_principles.yaml

```
@@ -6,8 +6,8 @@
 data_acquisition:
   display_name: Data acquisition
   description: |
-    A continuous uninterrupted block of time during which a brain scanning instrument was acquiring data according to
-    particular scanning sequence/protocol.
+    A continuous uninterrupted block of time during which an electromyography recording device was acquiring data,
+    or a brain scanning instrument was acquiring data according to particular scanning sequence/protocol.
```

```suggestion
    A continuous uninterrupted block of time during which a brain scanning instrument or an electromyography recording device was acquiring data according to
    particular scanning sequence/protocol.
```


don't put EMG in the front.